.github/workflows/npm-publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```

Comment:
what's up with the array of versions instead of just 1? Not saying it's wrong just curious how that works and why we do it

Reply:
It's testing against an array of versions since there are multiple projects with different node versions that use this library